common/on-chip-calib.cpp

```diff
@@ -1507,7 +1510,11 @@ namespace rs2
             try
             {
                 if (action == RS2_CALIB_ACTION_FL_CALIB)
+                {
                     calibrate_fl();
+                    _sub_color->stream_enabled = sub_color_restored_stream_enabled;
+                    _sub_color->stream_enabled[_uid_color]=true;
```

Comment:
I will let @OhadMeir review but as a general rule I can say:
1. Please keep alignment for coding style. `a=b` vs `a = b`
2. Please always add comments if what you are doing is not trivial
3. The map name can be improved to state what it maps
`std::map<int, bool> sub_color_restored_stream_enabled;`  currently it's hard to understand what this maps